### DISCUSSION TITLE
Widget Initial

### DISCUSSION THREAD

**Discussion by guilhermepiccioni** [stefanfoulis/django-phonenumber-field#503] [Q&A]
Good Morning,

I would like to know how do I start with the United Kingdom code, I put UK and it doesn't find +44.
`widgets = {
            'password': forms.PasswordInput(),
            'phone': PhoneNumberPrefixWidget(initial='UK'),
        }`

**Comment by amateja** [comment]
Hi,

You are initializing `PhoneNumberPrefixWidget` properly. Please share a bit more to let us understand your issue better.

https://github.com/stefanfoulis/django-phonenumber-field/blob/022dccc3caffe3aba0ea8388fb3d4a2201f2f0fb/tests/test_widgets.py#L29-L32

Regards
Andrzej

**Reply by guilhermepiccioni** [reply]
Good morning, in the code, when I put CN, BR... they work perfectly, it starts with the correct country, but when I put UK it doesn't look for anything.

my forms.py

```
class UserForm(forms.ModelForm):
    objects = None

    class Meta:
        model = NewAccount
        widgets = {
            'password': forms.PasswordInput,
            'phone': PhoneNumberPrefixWidget(initial='UK'),
        }
        fields = '__all__'

**Reply by francoisfreitag** [reply]
Thanks for the details. That’s because the region is named 'GB', not 'UK'.

```python
>>> import phonenumbers.phonenumberutil
>>> phonenumbers.phonenumberutil.COUNTRY_CODE_TO_REGION_CODE[44]
('GB', 'GG', 'IM', 'JE')
```
